docs/content/_index.md

```diff
@@ -5,16 +5,33 @@ date: 2018-02-11T16:52:23-05:00
 
 <img src="/banner.png" width="600" alt="Athens Logo"/>
 
+## Athens is a Server for Your Go Packages
 
-## Welcome to Athens
+Welcome, Gophers! We're looking forward to introducing you to Athens.. 
 
-Athens is the name of the combined project that includes a global registry for [Go Modules](https://github.com/golang/go/wiki/Modules) and a stand-alone proxy server that can be deployed on-premise to storage and control available Go modules for your organization. [Like what you hear? Try Athens Now!](/try-out)
+On this site, we document Athens in detail. We'll teach you what it does, why it matters, what you can do with it, and how you can run it yourself. Below is a brief summary for you.
+
+#### What Does Athens Do?
+
+Athens provides a server for [Go Modules](https://github.com/golang/go/wiki/Modules) that you can run. It serves public code and your private code for you, so you don't have to pull directly from a version control system (VCS) like GitHub or GitLab
+
+#### Why Does it Matter?
+
+Previously, the Go community has had lots of problems with libraries disappearing or changing without warning. If someone changes their library, your app can break overnight - oh no! Athens solves these problems by copying code from VCS's into _immutable_ storage.
```

Comment:
It would help to clarify "change their library". I believe we are referring to removing it entirely from VCS or force pushing tags that you were relying upon. But the wording here makes me also wonder if we are referencing "living dangerously off of the tip of master" too?